### DISCUSSION TITLE
Output and input .jsonl file for speciflic chat section

### DISCUSSION THREAD

**Discussion by Lostsite** [lobehub/lobehub#3375] [Ideas]
Sorry for bothering you guys once again. I just have another great idea about the project:

If we can export and import individual chat secession in a .jsonl file? The Openai can take .jsonl and do the batch process, and people can easily use this feature in Openai's dashboard, no program skill required. Since We can export the entire database in a .json file, it shouldn't be hard to import and export individual chat sessions. 

Here is Openai's document: https://platform.openai.com/docs/guides/batch/model-availability

The benefits: 
1. Reduce cost if you want to process a lot of text
2. Bypass rate limit (Openai has different rate limits for batch)
3. Possible to import individual chat in a more read-able way from other Gui (without importing the whole database).

I know you might be super busy, so, just a thought..... 

Best wishes

**Comment by arvinxx** [comment]
I think export can be easier. I can try to make it after the knowledge base feature
